draft-ietf-taps-interface.md

```diff
@@ -1515,18 +1518,13 @@ Attempts to clone a Connection can result in a CloneError:
 Connection -> CloneError<reason?>
 ~~~
 
-The Connection Property "Priority" operates on entangled Connections as in {{msg-priority}}:
-when allocating available network
+The Connection Priority Connection Property operates on entangled Connections 
```

Comment:
Bikeshed alert: can we just call it the "Priority Connection Property" to go with the "Priority Message Property"?